gunicorn/workers/gaiohttp.py

```diff
@@ -5,13 +5,23 @@
 
 import sys
 
-if sys.version_info >= (3, 3):
+from gunicorn import util
+
+if sys.version_info >= (3, 4):
     try:
         import aiohttp  # NOQA
     except ImportError:
         raise RuntimeError("You need aiohttp installed to use this worker.")
     else:
-        from gunicorn.workers._gaiohttp import AiohttpWorker
+        try:
+            from aiohttp.worker import GunicornWebWorker as AiohttpWorker
+        except ImportError:
+            from gunicorn.workers._gaiohttp import AiohttpWorker
+
+        util.warn(
```

Comment:
I think this kind of thing should be using Python's inbuilt [warning](https://docs.python.org/2/library/warnings.html)
logging facilities, with a [DeprecationWarning](https://docs.python.org/2/library/exceptions.html#exceptions.DeprecationWarning). This may be a separarte issue if there are other similar depreciation warnings.

Reply:
I made #1590/#1591 to use warnings in the other two places.